src/domain/events/detail/EventShow.tsx

```diff
@@ -141,6 +141,11 @@ const EventShow: FunctionComponent = (props: any) => {
         <Tab label={translate('events.show.tab.label')}>
           <LanguageTabs selectedLanguage={language} onSelect={selectLanguage} />
           <ImageField source="image" />
+          <TextField
+            source={`translations.${language}.imageAltText`}
+            label={translate('events.fields.imageAltText.label')}
```

Comment:
For React Admin component labels, you can do this just like `label="events.fields.imageAltText.label"` (yes, Kukkuu admin does not use that everywhere ATM 🙂  )

Reply:
Fixed in #32